RetrospectiveExtension.Frontend/tsconfig.json

```diff
@@ -13,7 +13,9 @@
     "noImplicitReturns": true,
     "noImplicitThis": true,
     "noImplicitAny": true,
-    "strictNullChecks": false
+    "strictNullChecks": false,
+    "allowSyntheticDefaultImports": true,
```

Comment:
both these two flags are needed since the component `AppInsightsErrorBoundary` found in  `@microsoft/applicationinsights-react-js` uses  a default import in its definition
`import React from "react";` this is not allowed unless we enable the `allowSyntheticDefaultImports` flag. Note that we can't change the `AppInsightsErrorBoundary` component file since it is auto installed by npm. If we owned that component the solution would have been trivial and we could have changed that statement to
`import * as React from "react";`

Reply:
Thanks for your info. It's really helpful for reviewers to understand the reason behind the changes.